paths/repo_syncs/index.yaml

```diff
@@ -0,0 +1,46 @@
+---
+summary: Get Repo Syncs
+description: Lists all Repo Syncs from an account
+operationId: repo_sync/index
```

Comment:
i think our convention is to list `list` as the operation id for the index endpoint